dhis-2/dhis-api/src/main/java/org/hisp/dhis/trackedentity/TrackedEntityInstanceQueryParams.java

```diff
@@ -203,6 +203,12 @@
      */
     private List<String> orders;
 
+    /**
+     * Indicates whether the query is run by Synchronization job or not. If yes, then the attributes with
+     * flag skipsynchronization set to true will not be included in the sync payload
+     */
+    private boolean synchronizationTask;
```

Comment:
Let us keep this variable a bit more descriptive, I suggest:

boolean excludeSkipSynchronizationItems;